palantir-java-format/src/main/java/com/palantir/javaformat/java/Main.java

```diff
@@ -130,7 +132,7 @@ private int formatFiles(CommandLineOptions parameters, JavaFormatterOptions opti
             }
         }
 
-        for (Map.Entry<Path, Future<String>> result : results.entrySet()) {
+        for (Entry<Path, Future<String>> result : results.entrySet()) {
```

Comment:
Was this intentional? I'm surprised this didn't trigger error-prone [BadImport](https://errorprone.info/bugpattern/BadImport) 

```suggestion
        for (Map.Entry<Path, Future<String>> result : results.entrySet()) {
```

Reply:
`"Entry"` isn't on BadImport's list: https://github.com/google/error-prone/blob/4165ecdc3d11a41b989d08e324de6fbc944e95ae/core/src/main/java/com/google/errorprone/bugpatterns/BadImport.java#L73-L84

Maybe it should be?

Reply:
Wow, looks like Entry was removed from the BadImport list 3 years ago, but my brain has already been conditioned to qualify Entry.

https://github.com/google/error-prone/commit/38a0e4b79ca04b20543ac8051b074c91def841f9